node/config.go

```diff
@@ -122,13 +123,13 @@ type DatabaseConfig struct {
 
 // Config contains all of the configurations of an Avalanche node.
 type Config struct {
-	HTTPConfig          `json:"httpConfig"`
-	IPConfig            `json:"ipConfig"`
-	StakingConfig       `json:"stakingConfig"`
-	genesis.TxFeeConfig `json:"txFeeConfig"`
-	StateSyncConfig     `json:"stateSyncConfig"`
-	BootstrapConfig     `json:"bootstrapConfig"`
-	DatabaseConfig      `json:"databaseConfig"`
+	HTTPConfig       `json:"httpConfig"`
+	IPConfig         `json:"ipConfig"`
+	StakingConfig    `json:"stakingConfig"`
+	fee.StaticConfig `json:"txFeeConfig"`
```

Comment:
I kept the json tag for backward compatibility, not sure if we want to change it  too @StephenButtolph